tests/integration_tests/sqllab_tests.py

```diff
@@ -523,8 +531,16 @@ def test_sqllab_viz_bad_payload(self):
             "chartType": "dist_bar",
             "schema": "superset",
             "columns": [
-                {"is_dttm": False, "type": "STRING", "name": f"viz_type_{random()}"},
-                {"is_dttm": False, "type": "OBJECT", "name": f"ccount_{random()}"},
+                {
+                    "is_dttm": False,
+                    "type": "STRING",
+                    "column_name": f"viz_type_{random()}",
```

Comment:
It's great that we're aligning the schemas here, but I would also consider this a breaking change, so let's hold of on merging this until 3.0.